app/api/api_v1/endpoints/student_router.py

```diff
@@ -33,17 +33,17 @@ async def list_students(
     students = await StudentService(db).list_students()
     return students
 
-@router.post("/students", response_model=StudentSchema)
-async def create_student_with_autogen_password(
-    *,
-    db: Session = Depends(get_db),
-    perm: None = Depends(PermissionDependency(StudentCreatePermission)),
-    student_body: CreateStudentBody
-):
-    student = await StudentService(db).create_student(
-        **student_body.dict()
-    )
-    return student
+# @router.post("/students", response_model=StudentSchema)
```

Comment:
Is this temporary in the PR? If not can you add another comment above explaining why it's commented out?

Reply:
Would you rather a comment or deletion ?

Reply:
I'd delete it if we don't have an explicit plan to use it later. The comment above a block of commented out code is just something we should do when we can't delete it because we plan to use it.